### ISSUE TITLE
Updates for the Account Reference Values report

### ISSUE THREAD

**Issue by jmcameron** [Third-Culture-Software/bhima#6163]
Finance > Report > Account Reference Values

**Potential problem:** With the Vanga dataset no selection of fiscal year and period produced non-zero values.   This could be a configuration issue and should be checked with other production databases.  Also, when a new Fiscal Year is selected, there are client-side errors.  This should be investigated further.

**Suggestions**
- Add multi-currency support
- Although you can select the period, it seems odd.  It actually shows the report for the fiscal year through that period.  A first time user would expect it to show the data for only that period (based on the title).   So perhaps the title above the Period selection field should be "Final Period" or something similar.

**Comment by jniles** [comment]
Yes, the Account References aren't used at Vanga.  They are a way of creating the OHADA Bilan, which Vanga never actually embraced OHADA formally.  They are used at IMCK, however.
